steps/browserstack-bitrise-xcui-step/1.0.0/step.yml

```diff
@@ -0,0 +1,222 @@
+title: BrowserStack App Automate - XCUITest
+summary: |
+  Run your XCUITest tests on BrowserStack App Automate
+description: |
+  Run your XCUITest tests on BrowserStack App Automate. This step collects the built IPA from $BITRISE_IPA_PATH  and test suite from `$BITRISE_BUNDLE_PATH` Environment Variables
+website: https://github.com/browserstack/browserstack-bitrise-xcui-step
+source_code_url: https://github.com/browserstack/browserstack-bitrise-xcui-step
+support_url: https://github.com/browserstack/browserstack-bitrise-xcui-step/issues
+published_at: 2022-08-16T20:11:30.271023+05:30
+source:
+  git: https://github.com/browserstack/browserstack-bitrise-xcui-step.git
+  commit: cd90e8eafe9e9130ea792e98b76b9e79eedde467
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/browserstack/browserstack-bitrise-xcui-step
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- browserstack_username: null
+  opts:
+    description: Username of the BrowserStack account
+    is_required: true
+    is_sensitive: true
+    summary: Username for the BrowserStack account
+    title: BrowserStack username
+- browserstack_accesskey: null
+  opts:
+    description: Access Key of the BrowserStack account
+    is_required: true
+    is_sensitive: true
+    summary: Accesskey for the BrowserStack account
+    title: BrowserStack accesskey
+- app_ipa_path: $BITRISE_IPA_PATH
+  opts:
+    description: Path of the app (.ipa) file
+    is_expand: true
+    is_required: true
+    summary: Path to the app (.ipa) file
+    title: iOS app under test
+- opts:
+    description: Path of the output bundle (.app) file. Add either the $BITRISE_TEST_BUNDLE_PATH
+      (generated from the "Xcode Build for testing for iOS" step) or the absolute
+      path of the output bundle file
+    is_expand: true
+    is_required: true
+    summary: Path of the output bundle file
+    title: XCUI test suite
+  xcui_test_suite: $BITRISE_TEST_BUNDLE_PATH
+- devices_list: null
+  opts:
+    description: "Name of one or more device-OS combination in new line \n For example:\n
+      \n `iPhone 11-13`\n \n `iPhone XS-15`\n \n [View supported device-OS combinations](https://www.browserstack.com/list-of-browsers-and-platforms/app_automate)\n"
+    is_expand: true
+    is_required: true
+    summary: Name of one or more device-OS combination in new line from the [device
+      list](https://www.browserstack.com/list-of-browsers-and-platforms/app_automate).
```

Comment:
Please don't use line breaks to wrap long lines, the UI will automatically wrap long lines.

This applies to all the other linebreaks in this file.